### ISSUE TITLE
Improve documentation to connect to remote kernel from Windows

### ISSUE THREAD

**Issue by ccordoba12** [spyder-ide/spyder-docs#95]
I found this article

https://medium.com/@mazzine.r/how-to-connect-your-spyder-ide-to-an-external-ipython-kernel-with-ssh-putty-tunnel-e1c679e44154

that explains an alternative method to connect to remote kernels on Windows when Paramiko doesn't work. Perhaps we should include a reference to it in our docs.

**Comment by CAM-Gerlach** [comment]
Thanks @ccordoba12 . After I get this big proposal in today, I should have some time this weekend to address my big backlog of docs, etc. issues.

**Comment by juanis2112** [comment]
@CAM-Gerlach can you take a look at this?

**Comment by CAM-Gerlach** [comment]
Yeah, sure. Maybe next weekend after I do the updates to the install guide.

**Comment by ccordoba12** [comment]
More on this topic:

* https://github.com/spyder-ide/spyder/issues/13133#issuecomment-651192994
* https://github.com/spyder-ide/spyder/issues/10240

**Comment by ccordoba12** [comment]
Also, the variable explorer is not working for remote kernels started through ssh. Still investigating why.

**Comment by CAM-Gerlach** [comment]
@juanis2112 Should I do this before or after you do your PR on the IPython console?

**Comment by CAM-Gerlach** [comment]
@ccordoba12 Since paramiko was added as a dependency, is this still necessary beyond what's done in #213 ?

**Comment by CAM-Gerlach** [comment]
@ccordoba12 I believe whatever of this wasn't mooted by adding Parmiko as a dependency on Windows (the original use case for this) should be covered by our detailed, step by step illustrated guide in the IPython Kernel docs as added in PR #213 . If there are specific future improvements that should be made there, that might be better handled in a new issue. As such, I'll go ahead and close this if I don't hear anything in a week.
